keel-sql/src/main/resources/db/changelog/20200930-environment-check-lock.yml

```diff
@@ -0,0 +1,29 @@
+databaseChangeLog:
+- changeSet:
+    id: environment-check-lock
+    author: fletch
+    changes:
+    - addColumn:
+        tableName: delivery_config_last_checked
+        columns:
+        - column:
+            name: locked_by
+            type: varchar(64)
+            constraints:
+              nullable: true
+        - column:
+            name: locked_at
+            type: timestamp(3)
```

Comment:
I think we've been using `timestamp(6)` as a standard?

Reply:
No, that would be nanosecond precision. `timestamp(3)` is millisecond.